src/pages/SubmissionConfig.test.js

```diff
@@ -1566,6 +1566,79 @@ describe('SubmissionConfig inputs/outputs definitions', () => {
     within(cells2[3]).getByDisplayValue('target_workflow_1_file_output')
   })
 
+  it('should set output variable names when set defaults button is clicked', async () => {
+    const mockRunSetResponse = jest.fn(() => Promise.resolve(runSetResponse))
+    const mockMethodsResponse = jest.fn(() => Promise.resolve(methodsResponse))
+    const mockSearchResponse = jest.fn((_, recordType) => Promise.resolve(searchResponses[recordType]))
+    const mockTypesResponse = jest.fn(() => Promise.resolve(typesResponse))
+
+    await Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runSets: {
+            getForMethod: mockRunSetResponse
+          },
+          methods: {
+            getById: mockMethodsResponse
+          }
+        },
+        Wds: {
+          search: {
+            post: mockSearchResponse
+          },
+          types: {
+            get: mockTypesResponse
+          }
+        }
+      }
+    })
+
+    render(h(SubmissionConfig))
```

Comment:
TOL based on our conversation yesterday... are we testing the submission config page as a whole or the `OutputsTable` as a component? I would guess based on the scope of the changes that maybe these would be better as component unit tests?